com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl

```diff
@@ -26,6 +26,20 @@ struct InputData
     half3   bakedGI;
     float2  normalizedScreenSpaceUV;
     half4   shadowMask;
+
+    half3   normalTS;
```

Comment:
This will cause some API breakage due to us adding more settings to InputData, we need to take another approach, cc @lukaschod

Reply:
This could be #ifdef'd like the parameters below, then it would only appear in shaders which required the feature. Would that solve the issue? Or is there more to it than that?

Reply:
`normalTS` is now removed from InputData.